CODEOWNERS

```diff
@@ -26,8 +26,27 @@
 /bgp/ @cilium/sig-bgp
 /cmd/ @cilium/cli
 /clustermesh/ @cilium/sig-clustermesh
-/connectivity/ @cilium/cli
+/connectivity/ @cilium/ci-structure
+/connectivity/check/ipcache.go @cilium/ipcache
+/connectivity/check/metrics*.go @cilium/metrics
```

Comment:
Errors are reported here. Those two teams don't seem to have access to this repository.

Reply:
I believe this should be resolved now.